crates/optimism/node/src/engine.rs

```diff
@@ -144,13 +144,20 @@ where
 
 impl<Types, P> EngineValidator<Types> for OpEngineValidator<P>
 where
-    Types: PayloadTypes<PayloadAttributes = OpPayloadAttributes, ExecutionData = OpExecutionData>,
+    Types: PayloadTypes<
+        PayloadAttributes = OpPayloadAttributes,
```

Comment:
can we refer to payload attributes from `Types` here too?
```suggestion
        PayloadAttributes = <Types as PayloadTypes>::PayloadAttributes,
```

Reply:
not quite because we do actually ne opattributes here for validation